app/views/root/index.html.erb

```diff
@@ -62,7 +62,7 @@
             <ol class="categories-list">
               <li>
                 <h3><a href="/register-to-vote?source=homepage-category">Register to vote</a></h3>
-                <p>Register to vote in elections and referendums. It takes less than 5 minutes.</p>
+                <p>You need to be registered to vote in the EU referendum. It takes less than 5 minutes.</p>
```

Comment:
Doesn't this imply that you _do_ need to register to vote for the EU referendum specifically?